sns_ik_lib/matlab/snsIk_vel_rr_cs.m

```diff
@@ -0,0 +1,188 @@
+function [dq, s, sCS, exitCode] = snsIk_vel_rr_cs(dqLow, dqUpp, dx, dqCS, J)
```

Comment:
Please add the same additional comments as I suggested for the acceleration solver.